src/components/Dashboard/index.js

```diff
@@ -331,7 +333,7 @@ const Dashboard = ({ background = "white", textColor = "black" }) => {
                     error={nameValidateError}
                     label="User Name"
                     variant="outlined"
-                    placeholder={displayName ? displayName : ""}
+                    placeholder={displayName ? displayName : "User Name"}
```

Comment:
keep it as `displayName && "User Name"` instead

Reply:
@Deeks900 Keep it as `displayName || "User Name"` 

it just removes extra complexity

Reply:
I got this. Thanks for clarifying

Reply:
In this updated commit I have done those new changes so that in case a user signs up using google and provides a display name then that name will be displayed as a value, not like a placeholder.
Previously it was getting displayed like this:
![image](https://user-images.githubusercontent.com/66011090/206141832-c848336a-4c82-4ed0-af92-f7ecc468a96e.png)

Now it will be shown as a value:
![image](https://user-images.githubusercontent.com/66011090/206141580-2d42d15a-5e00-46cb-85b6-afaaf949c5b5.png)

Reply:
greatt